Samples/Forms/com.sentry.ovenlibrary/AndroidProject/OvenLibrary/build.gradle

```diff
@@ -0,0 +1,50 @@
+plugins {
+    id 'com.android.library'
+}
+
+android {
```

Comment:
Why is the project on `Samples/Forms`? Sounds like an Android specific thing instead

Reply:
I plan in the future to move all Xamarin.Forms to the Forms folder while leaving other projects under the Samples folder (like tvOS , Xamarin.Droid, ...)